steps/version-extractor-android/0.0.1/step.yml

```diff
@@ -0,0 +1,56 @@
+title: Get versionName and versionCode on Android
+summary: |
+  Extract the versionName and the versionCode on an Android project using a Gradle task
+description: |
+  Extracts the Name version and the Code version from the Android project. It is useful if the versionName and versionCode in `build.gradle` or `build.gradle.kts` are defined somewhere else. Therefore, this step uses a Gradle task to print these two values.
+website: https://github.com/vladimirpetrovski/bitrise-step-version-extractor-android
+source_code_url: https://github.com/vladimirpetrovski/bitrise-step-version-extractor-android
+support_url: https://github.com/vladimirpetrovski/bitrise-step-version-extractor-android/issues
+published_at: 2020-05-02T17:57:15.726671+02:00
+source:
+  git: https://github.com/vladimirpetrovski/bitrise-step-version-extractor-android.git
+  commit: 7e9f33fe3853cf49c8a8880335e37a3ffbfd57ea
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_gradle_path: $BITRISE_SOURCE_DIR/app/build.gradle
```

Comment:
Step description says about kts files. However, script seems to support only Groovy files.

Reply:
For now, this script supports only Groovy files. If it a requirement from you to support .kts files, please let me know otherwise I'll implement that feature in later versions. Check https://github.com/bitrise-io/bitrise-steplib/pull/2470. Thanks!